README.md

```diff
@@ -258,68 +300,54 @@ tests:
+used by external applications to present metrics and data for others to see into. An example of
+this is they are used to present data in [TestGrid Dashboards][TestGrid Dashboard].
 
 ### CI/CD Job Results Database
 
-We have provisioned an AWS RDS Postgres database to store information about our CI jobs and the tests that they execute. We used to store our data only within prometheus, but prometheus's timeseries paradigm prevented us from being able to express certain queries (even simple ones like "when was the last time this test failed").
+We have provisioned an AWS RDS Postgres database to store information about our CI jobs
+and the tests that they execute. We used to store our data only within prometheus,
+but prometheus's timeseries paradigm prevented us from being able to express certain
+queries (even simple ones like "when was the last time this test failed").
 
-The test results database (at time of writing) stores data about each job and its configuration, as well as about each test case reported by the Junit XML output of the job.
+The test results database (at time of writing) stores data about each job and its configuration,
+as well as about each test case reported by the Junit XML output of the job.
 
-This data allows us to answer questions about frequency of job/test failure, relationships between failures, and more. The code responsible for managing the database can be found in the [`./pkg/db/`](https://github.com/openshift/osde2e/tree/cfd38c75532274d619840ad505c1232881eb417a/pkg/db) directory, along with a README describing how to develop against it.
+This data allows us to answer questions about frequency of job/test failure,
+relationships between failures, and more. The code responsible for managing the
+database can be found in the [`./pkg/db/`](pkg/db) directory,
+along with a README describing how to develop against it.
 
-#### Database usage from OSDe2e
+### Database usage from OSDe2e
 
-Because `osde2e` runs a a cluster of parallel, ephemeral prow jobs, our usage of the database is unconventional. We have to write all of our database interaction logic with the understanding that any number of other prow jobs could be modifying the data at the same time that we are.
+Because `osde2e` runs a a cluster of parallel, ephemeral prow jobs, our usage of
+the database is unconventional. We have to write all of our database interaction
+logic with the understanding that any number of other prow jobs could be modifying
+the data at the same time that we are.
 
-We use the database to generate alerts for the CI Watcher to use, and we follow this algorithm to generate those alerts safely in our highly-concurrent usecase (at time of writing, implemented [here](https://github.com/openshift/osde2e/blob/cfd38c75532274d619840ad505c1232881eb417a/pkg/e2e/e2e.go#L1029)):
+We use the database to generate alerts for the CI Watcher to use, and we follow
```

Comment:
This CI Watcher snippet can be removed. That role was deprecated. 
We should have a conversation on that Db instance at some point as well*

Reply:
Thanks for pointing that out, I will remove it. :+1: to having a conversation about the DB instance.